<ui version="4.0" >
 <class>AddCardsDlg</class>
 <widget class="QDialog" name="AddCardsDlg" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>330</width>
    <height>176</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Add cards</string>
  </property>
  <layout class="QHBoxLayout" >
   <item>
    <layout class="QVBoxLayout" >
     <item>
      <layout class="QGridLayout" >
       <item row="0" column="0" >
        <widget class="QLabel" name="label_2" >
         <property name="text" >
          <string>Card type:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" >
        <widget class="QComboBox" name="card_types" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Fixed" hsizetype="MinimumExpanding" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="1" column="0" >
        <widget class="QLabel" name="label" >
         <property name="text" >
          <string>Category:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" >
        <widget class="QComboBox" name="categories" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Fixed" hsizetype="MinimumExpanding" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="editable" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="grades" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="MinimumExpanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize" >
        <size>
         <width>310</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>16777215</width>
         <height>66</height>
        </size>
       </property>
       <property name="title" >
        <string>Select initial grade:</string>
       </property>
       <layout class="QVBoxLayout" >
        <item>
         <layout class="QHBoxLayout" >
          <item>
           <widget class="QPushButton" name="grade_0_button" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Fixed" hsizetype="MinimumExpanding" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize" >
             <size>
              <width>40</width>
              <height>0</height>
             </size>
            </property>
            <property name="text" >
             <string>0</string>
            </property>
            <property name="shortcut" >
             <string>Ctrl+0</string>
            </property>
            <property name="autoDefault" >
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="grade_1_button" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Fixed" hsizetype="MinimumExpanding" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize" >
             <size>
              <width>40</width>
              <height>0</height>
             </size>
            </property>
            <property name="text" >
             <string>1</string>
            </property>
            <property name="shortcut" >
             <string>Ctrl+1</string>
            </property>
            <property name="autoDefault" >
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="Line" name="line" >
            <property name="orientation" >
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="grade_2_button" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Fixed" hsizetype="MinimumExpanding" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize" >
             <size>
              <width>40</width>
              <height>0</height>
             </size>
            </property>
            <property name="text" >
             <string>2</string>
            </property>
            <property name="shortcut" >
             <string>Ctrl+2</string>
            </property>
            <property name="autoDefault" >
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="grade_3_button" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Fixed" hsizetype="MinimumExpanding" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize" >
             <size>
              <width>40</width>
              <height>0</height>
             </size>
            </property>
            <property name="text" >
             <string>3</string>
            </property>
            <property name="shortcut" >
             <string>Ctrl+3</string>
            </property>
            <property name="autoDefault" >
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="grade_4_button" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Fixed" hsizetype="MinimumExpanding" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize" >
             <size>
              <width>40</width>
              <height>0</height>
             </size>
            </property>
            <property name="text" >
             <string>4</string>
            </property>
            <property name="shortcut" >
             <string>Ctrl+4</string>
            </property>
            <property name="autoDefault" >
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="grade_5_button" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Fixed" hsizetype="MinimumExpanding" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize" >
             <size>
              <width>40</width>
              <height>0</height>
             </size>
            </property>
            <property name="text" >
             <string>5</string>
            </property>
            <property name="shortcut" >
             <string>Ctrl+5</string>
            </property>
            <property name="autoDefault" >
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" >
       <item>
        <widget class="QPushButton" name="preview_button" >
         <property name="text" >
          <string>&amp;Preview</string>
         </property>
         <property name="autoDefault" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>101</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="exit_button" >
         <property name="text" >
          <string>E&amp;xit</string>
         </property>
         <property name="autoDefault" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
